plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -604,6 +605,15 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
                 ImmutableList.of());
     }
 
+    @Override
+    public Optional<Type> getSupportedType(ConnectorSession session, Map<String, Object> tableProperties, Type type)
+    {
+        if (type instanceof TimestampType) {
+            return Optional.of(TIMESTAMP_MICROS);
```

Comment:
Could you add a test that ensures `timestampNtz` reader & writer features exist in transaction log's protocol entry? I manually confirmed it's stored correctly though. The connector has a special logic for timestamp type. You can refer to `TestDeltaLakeBasic#testTrinoCreateTableWithTimestampNtz()`